testsuite/unittests/test_connect.sli

```diff
@@ -177,3 +177,52 @@ ResetKernel
   Last /target get 6 eq assert_or_die
 
 } forall
+
```

Comment:
maybe it would make sense to get all devices from the modeldict and loop through all possible combinations instead of listing them manually here? this would avoid the need to add new tests here upon adding a new device and make sure that we do not forget a certain combination.